index.html

```diff
@@ -3555,6 +3555,21 @@ <h3>
 Signatures on <a>DID documents</a> are optional. <a>DID method</a>
 specifications SHOULD explain and specify their implementation if applicable.
         </p>
+
+        <div class="note" role="note" id="issue-container-generatedID-8">
+          <div role="heading" class="note-title marker" id="h-note-7" 
+          aria-level="4"><span>Note</span></div><p class="">
+Through the use of digital signatures a controller can assert proof of control 
+which can be used in a variety of different contexts. By asserting proof of 
+control of the identifier, the controller is asserting proof of ownership of 
+the public key which may not be the same as the subject of the DID. However, 
+it's up to the verifier to determine if the usage of digital signatures is 
```

Comment:
Avoid use of contractions in official specifications; use more formal language. If you find yourself using a large number of contractions, it is typically a sign that your sentence needs to be reworded to be simpler.

Reply:
I would remove the language that proof of control is related to "proof of ownership". A major point of the language of "proof-of-control" is that ownership is a matter of law, not technology. So, only courts can establish actual ownership. Proof-of-control, on the other hand, is technically concrete. So, when someone steals your keys, they *can* generate proof-of-control, but they CANNOT establish proof of ownership. "Not your keys, not your coins" fails in the reverse: "Controlling the keys does not prove they are your coins." You *can* spend them--you have control--but you don't own them any more than the car thief who stole your keys to steal your car. They don't "own" your car, nor do they establish proof of ownership. They just control it.